service/src/test/java/uk/gov/hmcts/reform/fpl/service/removeorder/RemoveOrderServiceTest.java

```diff
@@ -115,46 +127,68 @@ void shouldMakeDynamicListOfMixedOrderTypes() {
 
     @Test
     void shouldUseGeneratedOrderRemovalActionWhenRemovingGeneratedOrder() {
```

Comment:
I don't think you need to test the specific actions. Those should be tested separately. you can assume that  orderRemovalActions.getAction(removableOrder) should return you the right action(a mock implementing the OrderRemovalAction)